docs/docs/events/builtin/extra/tracee_info.md

```diff
@@ -0,0 +1,27 @@
+# tracee_info
+
+## Intro
+
+tracee_info - An event that exports some relevant data of Tracee upon startup.
+
+## Description
+
+This event, created in user-mode during Tracee's initialization, is typically the first event emitted. It provides valuable metadata about Tracee's configuration and runtime environment, which can be helpful for event processing and troubleshooting.
+
+The event was created also with Tracee's File Source in mind, to provide information about how Tracee ran during the original capture.
+
+## Arguments
+
+* `boot_time`:`u64`[U] - the boot time of the system Tracee run in since epoch.
+* `start_time`:`u64`[U] - the Tracee time elapse since system boot. This indicates when the Tracee process started relative to the system boot time.
```

Comment:
```suggestion
* `start_time`:`u64`[U] - the time elapsed since Tracee started, relative to the system boot time. This indicates when the Tracee process started relative to the system boot time.
```